fawkes/fetch/splunk.py

```diff
@@ -0,0 +1,33 @@
+import json
+import splunklib.client as client
+import splunklib.results as results
+
+def fetch(review_channel):
+    reviews = []
+    try:
+        service = client.connect(
+            host=review_channel.host,
+            port=review_channel.port,
+            username=review_channel.username,
+            password=review_channel.password,
+            scheme="https",
+            basic=True,
+        )
+
+        query = 'search {} earliest=-{} latest={}'.format(review_channel.query, review_channel.earliest, review_channel.latest)
+
+        stream = service.jobs.export(query)
+
+        reader = results.ResultsReader(stream)
+        for result in reader:
+            if isinstance(result, dict):
+                review = json.loads(result["_raw"])
+                reviews.append(review)
+            elif isinstance(result, results.Message):
```

Comment:
What exactly is this scenario ? Why are we just printing the message in this case ?

Reply:
We dont need this else block. removed it